layout/composite.c

```diff
@@ -134,7 +134,7 @@ static m0_bcount_t composite_max_recsize(struct m0_layout_domain *dom)
 	return 0;
 }
 
-static const struct m0_layout_ops composite_ops;
+M0_UNUSED static const struct m0_layout_ops composite_ops;
```

Comment:
I verified that in main branch, the following variables and functions are already marked as unused code.

Did you regularly merge the main branch into the integration branch?
Please keep the integration update.